### PR TITLE
Add time zone documentation

### PR DESCRIPTION
**Pull request by mboussaa**
-Update JP documentation (e.g., figure)
-Add time zone documentation

### CODE REVIEW

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc [hunk 1]
```diff
@@ -114,54 +114,48 @@ The "Save Gantt" button will take a screenshot of the visible part of the Gantt
 
 Job Planner uses a  Calendar Definition to know how the job will be planned over the time. As shown on the example below, this definition is composed of 4 fields:
 
- - a description (saying what the cron expression means, when to use the Calendar Definition, etc.)
+ - a description (providing humain-readable description of the used cron expression, when is the Calendar Definition used, etc.)
```
Comment (mklkun) [inline src/docs/JobPlanner/references/JobPlannerServiceReference.adoc]: humain-readable => a humain-readable

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc [hunk 2]
```diff
@@ -114,54 +114,48 @@ The "Save Gantt" button will take a screenshot of the visible part of the Gantt
...
+ - a time zone to define a specific time zone for the future scheduled jobs depending on the region (Europe/Paris, US/Eastern, etc.)
```
Comment (mklkun) [inline src/docs/JobPlanner/references/JobPlannerServiceReference.adoc]: please check the repetition here "a time zone to define a specific time zone". At least, if the first used `time zone` in the sentence is a button or something refer to it using capitals.

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc [hunk 3]
```diff
@@ -114,54 +114,48 @@ The "Save Gantt" button will take a screenshot of the visible part of the Gantt
...
 ----
 
 === Description
 
-The description allows users who are not familiar with cron expressions to know when it will occur. It might also be used for other purpose, for example saying when to use a Calendar Definition.
+The description allows users who are not familiar with cron expressions to provide a humain-readable description to know when it occurs. It might also be used for other purpose, for example saying when is the Calendar Definition used.
+
+=== Time Zone
+
+The aim of the time zone is to make the user able to select a specific time zone for his future scheduled jobs depending on the region. The default time zone is the client time zone. The server time zone is always displayed on the top right corner. It indicates the time of the machine where ProActive instance is running.
```
Comment (mklkun) [inline src/docs/JobPlanner/references/JobPlannerServiceReference.adoc]: Same comment.

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc (hunk 2 above)
Comment (mboussaa) [reply src/docs/JobPlanner/references/JobPlannerServiceReference.adoc]: done

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc (hunk 1 above)
Comment (mboussaa) [reply src/docs/JobPlanner/references/JobPlannerServiceReference.adoc]: done

File: src/docs/JobPlanner/references/JobPlannerServiceReference.adoc (hunk 3 above)
Comment (mboussaa) [reply src/docs/JobPlanner/references/JobPlannerServiceReference.adoc]: fixed

**Pull request merged**